src/tasks/DockerRunTaskProvider.ts

```diff
@@ -82,10 +85,6 @@ export class DockerRunTaskProvider implements TaskProvider {
         if (!dockerRun.image) {
             throw new Error('No Docker image name was provided or resolved.');
         }
-
-        if (!dockerRun.containerName) {

```

Comment:
I could go either way on this. Upside, without this check users don't _have_ to specify container name. Downside, none of our debugging works without container name, so this serves as extra validation for that.

Reply:
@philliphoff In your mongo example, do you have to provide container name anyway?

Reply:
No, in the MongoDB example it was only necessary to set the network alias, not the name of the container.  This is potentially an advantage of having a "debugging" flag, where we can apply different validations in that case.